src/translations/sk.json

```diff
@@ -0,0 +1,41 @@
+{
+    "log_in": "Prihlásiť sa",
+    "log_out": "Odhlásiť sa",
+    "logged_in_as": "Prihlásený ako",
+    "logged_in": "Prihlásený",
```

Comment:
In English a passive tone makes sense, but I don't think it translates as well into Slovak, tbh.

What do you think of specifying who is logged in (as in, Logged in user), for example like so:
```suggestion
    "logged_in": "Prihlásený užívateľ",
```

Reply:
@iKristy Actually, I am not sure where exactly it shows up. It sounds like it would make more sense for "logged_in_as".

Reply:
It shows up here:

![image](https://user-images.githubusercontent.com/7912948/120668905-d7cb0380-c486-11eb-9f75-18072a6c82e9.png)

I think the title change there works a bit better.

Reply:
@iKristy Ok, fixed